source/extensions/omni.isaac.orbit/omni/isaac/orbit/terrains/terrain_importer.py

```diff
@@ -343,3 +343,11 @@ def _compute_env_origins_grid(self, num_envs: int, env_spacing: float) -> torch.
         env_origins[:, 1] = env_spacing * yy.flatten()[:num_envs] - env_spacing * (num_cols - 1) / 2
         env_origins[:, 2] = 0.0
         return env_origins
+
+    """
+    Dummy Functions.
+    """
+
+    def sample_new_targets(self, env_ids: torch.Tensor) -> torch.Tensor:
+        """Returns the environment origins as target locations."""
```

Comment:
```suggestion
    def sample_new_targets(self, env_ids: Sequence[int]) -> torch.Tensor:
        """Samples terrain-aware locations of flat patches to set spawn or target locations.
        
        Note:
             This is a dummy implementation that returns the environment origins as target locations. Please inherit the class and reimplement the function for specific terrain types.
         
         Args:
             env_ids: The environment indices to sample target locations for.
          
          Returns:
              The sampled target locations as (x, y, z, yaw). Shape is (N, 4).
         """
```

Reply:
Updated the docstring in [e075bf0](https://github.com/NVIDIA-Omniverse/orbit/pull/191/commits/e075bf0ecbb4655f38ad46d9d161ab729e09c558), it returns (x, y, z) of shape (N, 3) right?